src/SwedbankPay.Sdk/PaymentInstruments/Card/ICardPaymentRecurResponseDetails.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    /// <summary>
+    /// Transactional details about a recurring credit card payment.
+    /// </summary>
+    public interface ICardPaymentRecurResponseDetails

```

Comment:
If we add an `IIdentifiable` interface, we can declare all other interfaces as identifiable like this. Thoughts?

```suggestion
    public interface ICardPaymentRecurResponseDetails : IIdentifiable
```

Reply:
Sure